src/ClientHandle.cpp

```diff
@@ -1532,6 +1539,27 @@ void cClientHandle::HandleSlotSelected(Int16 a_SlotNum)
 
 
 
+void cClientHandle::HandleSpectate(const AString & a_PlayerUUID)
```

Comment:
Yes, which would split the code up. The problem with cClientHandle is that there is too much code in one module. And as for bugs, splitting the code up would make writing tests easier.